tests/atoms/gaia_data_layer.js

```diff
@@ -66,16 +67,17 @@ var GaiaDataLayer = {
 
   enableBluetooth: function() {
     var bluetooth = window.navigator.mozBluetooth;
-    if (!bluetooth.enabled) {
+    if (bluetooth.defaultAdapter.state != 'enabled') {
```

Comment:
Why? bluetooth.defaultAdapter.state != 'enabled' does cover all the states we don't want to have, no?